task/build-vm-image/0.1/build-vm-image.yaml

```diff
@@ -249,23 +263,23 @@ spec:
         if [ -f "/output/qcow2/disk.qcow2" ]; then
           echo -e "Found qcow2 image."
           # don't zip qcow2 images, it is already compressed.
-          buildah manifest add --arch $(arch) --os linux --artifact --artifact-type application/vnd.diskimage.qcow2.gzip $OUTPUT_IMAGE /output/qcow2/disk.qcow2
+          buildah manifest add --arch $(arch) --os linux --artifact --artifact-type application/vnd.diskimage.qcow2.gzip $OUTPUT_IMAGE /output/qcow2/disk.qcow2 --all
```

Comment:
These commands just add a single artifact. I don't think that we need `--all` here. That parameter is necessary when we are generating an image index of image references which are themselves image indexes.

Reply:
Makes sense. Removed.